netbox/dcim/models/cables.py

```diff
@@ -237,7 +237,10 @@ def save(self, *args, **kwargs):
                 if not termination.pk or termination not in b_terminations:
                     CableTermination(cable=self, cable_end='B', termination=termination).save()
 
-        trace_paths.send(Cable, instance=self, created=_created)
+        try:
+            trace_paths.send(Cable, instance=self, created=_created)
+        except ValidationError as e:
+            raise ValidationError(f'{e}')
```

Comment:
Why would we catch and re-raise the same exception? Was this meant to catch AssertionErrors?

Reply:
That should have been an assertion error.  I changed this code a few times and must have accidentally changed it to ValidationError before commiting.